README.adoc

```diff
@@ -122,16 +122,24 @@ QueryResource.java
 include::finish/query/src/main/java/io/openliberty/guides/query/QueryResource.java[]
 ----
 
+The [hotspot=systemLoad file=0]`systemLoad` endpoint first gets all the hostnames by calling [hotspot=getSystems file=0]`getSystems()`.
+Then it loops through each hostname and calls [hotspot=getSystem file=0]`getSystem()` on each one. 
+This returns a `CompletionStage` interface which is processed asynchronously in the [hotspot=thenAcceptAsync file=0]`thenAcceptAsync()` method. 
+
```

Comment:
Can we further clarify that these requests will be sent asynchronously?

Reply:
Updated last two sentences to 

```
Each hostname is asynchronously processed using the method [hotspot=getSystem file=0]`getSystem()` and the method [hotspot=thenAcceptAsync file=0]`thenAcceptAsync()` from the `CompletionStage` interface.
```

Reply:
Can we phrase it in terms of async requests will be sent to the client service for each hostname?